internal/welcome/welcome.go

```diff
@@ -33,11 +38,7 @@ const (
 // Create a welcome to Sugarcity.io Slack workspace message.
 func welcomeMessage(u string) string {
 
-	msg := fmt.Sprintf("Hi <@%s>, welcome to the Sugarcity.io Slack! :wave: :sugarcity-green:\n"+
-		"We are excited to have you on board with Mackay's greatest gathering of technologists and innovators! :rocket:\n"+
-		"It would be awesome if you could introduce yourself in the <#%s> channel so we can get to know you! :smile:\n"+
-		"We have a bunch of channels to checkout, like <#%s>, <#%s>, <#%s>, <#%s>, <#%s> and a load more. :tada:\n"+
-		"Our community is here to support you, so don't hesitate to ask questions or share your own knowledge. :muscle:\n", u, GeneralChannelID, CodeConvoChannelID, EventsChannelID, InfoSecChannelID, ITTechChannelID, GamingChannelID)
+	msg := fmt.Sprintf(message, u, GeneralChannelID, CodeConvoChannelID, EventsChannelID, InfoSecChannelID, ITTechChannelID, GamingChannelID)
```

Comment:
This is a good improvement.

As a potential future idea, maybe [text/template](https://pkg.go.dev/text/template) can be leveraged to give each placeholder value in welcome.txt a distinct identifier. This way they can be populated by name instead of assuming the order in which they appear.

Reply:
Oh that is nice! Thanks for pointing that out! Had no idea that was even a thing!